SkyrimNet-Zonos.py

```
@@ -105,15 +107,31 @@ async def generate_audio(model_choice, text, language, speaker_audio, prefix_aud
+    if text == "ping":
+       if IGNORE_PING is None:
+        IGNORE_PING = "pending"
+       else:
+          logger.info("Ping request received, sending silence audio.")
+          return "assets/silence_100ms.wav", seed
+
+    emotions = [e1, e2, e3, e4, e5, e6, e7, e8]
+
+    if speaker_audio is None:
+        speaker_audio = 'malecommoner'
+        logger.info(f'Requested: default {speaker_audio} with{f"out emotion." if "emotion" in unconditional_keys else f" emotion values: {emotions}"} Text: "{text}"')
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)